src/app/pages/home/layouts/projects/projects.component.html

```
@@ -1,5 +1,5 @@
-<section id="projects">
-  <div data-aos="fade-up" data-aos-duration="75">
+<section>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)